udata/core/organization/rdf.py

```diff
@@ -28,4 +31,52 @@ def organization_to_rdf(org, graph=None):
     o.set(RDFS.label, Literal(org.name))
     if org.url:
         o.set(FOAF.homepage, URIRef(org.url))
+
     return o
+
+
+def build_org_catalog(org, datasets, format=None):
+    graph = Graph(namespace_manager=namespace_manager)
+    org_catalog_url = url_for('organizations.rdf_catalog', org=org.id, _external=True)
+
+    catalog = graph.resource(URIRef(org_catalog_url))
+    catalog.set(RDF.type, DCAT.Catalog)
+    catalog.set(DCT.publisher, organization_to_rdf(org, graph))
+
+    for dataset in datasets:
+        catalog.add(DCAT.dataset, dataset_to_rdf(dataset, graph))
+    
+    if isinstance(datasets, Paginable):
```

Comment:
This can't be factorized with the main catalog?

Reply:
The dataset part is not exactly the same the main catalog add the owner or the organization to the dataset:
```
for dataset in datasets:
    rdf_dataset = dataset_to_rdf(dataset, graph)
    if dataset.owner:
        rdf_dataset.add(DCT.publisher, user_to_rdf(dataset.owner, graph))
    elif dataset.organization:
        rdf_dataset.add(DCT.publisher, organization_to_rdf(dataset.organization, graph))
    catalog.add(DCAT.dataset, rdf_dataset)
```

For the pagination part I thought about it but as the making of urls arguments depends on the model, for instance
```
first_url = url_for('organizations.rdf_catalog_format', org=org.id, page=1, **kwargs)
```
It might fall into something overly engineered and loss of the code readability to make it generic.
What do you think?